features/snake/gui.lua

```diff
@@ -11,10 +11,11 @@ local function show_gui_for_player(player)
         return
     end
 
-    local top = player.gui.top
-    if not top[main_button_name] then
-        top.add {type = 'button', name = main_button_name, caption = {'snake.name'}}
-    end
+    Gui.get_top_element(player, {
```

Comment:
I think this should be
```suggestion
    Gui.add_top_element(player, {
```

To test this you can uncomment `require 'features.snake.control'` in control.lua and then run the following command:
```
/c  local surface = game.player.surface local pos = game.player.position pos = {x = math.floor(pos.x + 2), y = math.floor(pos.y + 2)} local size = 15 local update_rate = 30 local max_food = 6 remote.call('snake', 'start_game', surface, pos, size, update_rate, max_food)
```